lab-miketreat/model/kid-toy.js

```diff
@@ -0,0 +1,16 @@
+'use strict';
+
+const debug = require('debug')('http:kid-toy');
+const uuid = require('node-uuid');
+
+module.exports = function(name, type, hazard, id) {
+  if(!name || !type) throw new Error('Invalid arguments');
+
+
+
+  this.name = name;
+  this.type = type;
+  this.hazard = hazard;
+  this.id = id || 1;
+  // this.id = uuid();
```

Comment:
This uuid module is what you want but with the `node-uuid` package (which has been deprecated) you want to call it like this: `uuid.v4()`, so that's probably why that line wasn't working for you.